src/App.js

```diff
@@ -0,0 +1,198 @@
+          i++
+        ) {
+          for (
+            let j = 0;
+            j < response.results[0].address_components[i].types.length;
+            j++
+          ) {
+            switch (response.results[0].address_components[i].types[j]) {
+              case "locality":
+                city = response.results[0].address_components[i].long_name;
+                break;
+              case "administrative_area_level_1":
+                state = response.results[0].address_components[i].long_name;
+                break;
+              case "country":
+                country = response.results[0].address_components[i].long_name;
+                break;
+            }
+          }
+        }
+        setAddress(city);
+      },
+      (error) => {
+        console.error(error);
+      }
+    );
+  };
+
+  return (
+    <div className="App">
+      <header className="App-header pt-5">
+        <div className="container">
+          <div className="wrapper pt-5">
+            <div className="row">
+              <div className="col-2"></div>
+              <div className="col-8">
+                <div className="input-group">
+                  <FaCrosshairs
+                    onClick={getLocation}
+                    className="icon-location"
+                  />
+                  <input
+                    type="text"
+                    className="form-control"
+                    placeholder="Enter Location..."
+                    onChange={(e) => setAddress(e.target.value)}
+                  />
+                </div>
+              </div>
+              <div className="col-2"></div>
+            </div>
+            <br />
+            {locationInfo ? (
+              <div className="container">
+                <div className="row">
+                  <div className="col-sm-12 col-xs-12 col-md-12 col-lg-6 col-xl-6 pt-5">
+                    <h1 className="teal-color big-text">
+                      {kelvinToFahrenheit(locationInfo.main.temp)}&deg; C
+                    </h1>
+                    <br />
+                    <ReactCountryFlag
+                      countryCode={locationInfo.sys.country}
+                      style={{
+                        fontSize: "2em",
+                        lineHeight: "3em",
+                        marginBottom: "1em",
+                      }}
+                      svg
+                    />
+                    <h2>
+                      <FaLocationArrow /> {locationInfo.name}
+                    </h2>
+                    <br />
+                    <h5>{countryData.country}</h5>
+                  </div>
+                  <div className="col-sm-12 col-xs-12 col-md-12 col-lg-6 col-xl-6 pt-5">
+                    <div className="weatherWrapper">
+                      <img
+                        src={`http://openweathermap.org/img/w/${locationInfo?.weather[0].icon}.png`}
+                        alt="weather status icon"
```

Comment:
formatting one line

Reply:
@Henshall when line length increases from 80 characters then readability gets compromised. To improve readability single line is separated into multiple lines. see industry standard code editor extension https://prettier.io/